src/components/BasicTable.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './BasicTable.scss';
```

Comment:
is css is imported into each component? and file exports a single html component?